### PR TITLE
added formatting features to live and pre-recorded transcriptions

### PR DESCRIPTION
**Pull request by briancbarrow**

### CODE REVIEW

File: src/types/liveTranscriptionOptions.ts [hunk 1]
```diff
@@ -100,9 +100,15 @@ export type LiveTranscriptionOptions = {
...
-   * @see https://developers.deepgram.com/api-reference/speech-recognition-api#operation/transcribeAudio/properties/numerals
+   * @see https://developers.deepgram.com/documentation/features/numerals/
    */
-  numerals?: boolean;
+  numbers?: boolean;
+
+  /**
+   * adds spaces between numbers in the transcript
+   */
+  numbers_space?: boolean;
```
Comment (shirgoldbird) [inline src/types/liveTranscriptionOptions.ts:110]: Should be plural: `numbers_spaces`

File: src/types/prerecordedTranscriptionOptions.ts [hunk 2]
```diff
@@ -107,7 +107,12 @@ export type PrerecordedTranscriptionOptions = {
    * 999,999 would be transcribed as 999999.
    * @see https://developers.deepgram.com/api-reference/speech-recognition-api#operation/transcribeAudio/properties/numerals
    */
-  numerals?: boolean;
+  numbers?: boolean;
```
Comment (shirgoldbird) [inline src/types/prerecordedTranscriptionOptions.ts:110]: Same comment here, we should keep `numerals` around for now

File: src/types/prerecordedTranscriptionOptions.ts [hunk 3]
```diff
@@ -107,7 +107,12 @@ export type PrerecordedTranscriptionOptions = {
...
+
+  /**
+   * adds spaces between numbers in the transcript
+   */
+  numbers_space?: boolean;
```
Comment (shirgoldbird) [inline src/types/prerecordedTranscriptionOptions.ts:115]: Plural: `numbers_spaces`

File: src/types/prerecordedTranscriptionOptions.ts (hunk 2 above)
Comment (shirgoldbird) [inline src/types/prerecordedTranscriptionOptions.ts:110]: Same comment here, we should keep `numerals` around for now

File: src/types/usageOptions.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ export type UsageOptions = {
   search?: boolean;
   redact?: boolean;
   alternatives?: boolean;
-  numerals?: boolean;
```
Comment (shirgoldbird) [inline src/types/usageOptions.ts:22]: Keep numerals

File: src/types/usageRequestDetail.ts [hunk 5]
```diff
@@ -24,8 +24,8 @@ export type UsageRequestDetail = {
       search?: boolean;
       redact?: boolean;
       alternatives?: boolean;
-      numerals?: boolean;
```
Comment (shirgoldbird) [inline src/types/usageRequestDetail.ts:27]: Keep numerals

**Review by shirgoldbird — CHANGES_REQUESTED** [review]
Thank you so much! Requested a few changes

File: src/types/liveTranscriptionOptions.ts [hunk 6]
```diff
@@ -182,4 +188,44 @@ export type LiveTranscriptionOptions = {
...
+  /**
+   * Indicates whether to convert times from written format (e.g., three oclock) to
+   * numerical format (e.g., 3:00).
+   * 	*/
+  times: boolean;
+
+  /**
+   * Combines dates and times into one formatting option format
+   */
+  datetime?: boolean;
```
Comment (shirgoldbird) [inline src/types/liveTranscriptionOptions.ts:214]: Can you remove all the `datetime` and `datetime_format` options? I realized this feature doesn't exist yet... 😅

File: src/types/liveTranscriptionOptions.ts [hunk 7]
```diff
@@ -100,9 +100,15 @@ export type LiveTranscriptionOptions = {
    * numerical format (e.g., 1). Deepgram can format numbers up to 999,999.
    * @remarks Converted numbers do not include punctuation. For example,
    * 999,999 would be transcribed as 999999.
-   * @see https://developers.deepgram.com/api-reference/speech-recognition-api#operation/transcribeAudio/properties/numerals
+   * @see https://developers.deepgram.com/documentation/features/numerals/
    */
-  numerals?: boolean;
```
[the remark this answers is not in the document]
Comment (briancbarrow) [reply src/types/liveTranscriptionOptions.ts:105]: So do we want to have both `numbers` and `numerals`?

**Review by shirgoldbird — APPROVED** [review]
LGTM! Thank you 🙏🏻

**Pull request merged**